packages/twenty-server/src/engine/metadata-modules/field-metadata/field-metadata.service.ts

```diff
@@ -716,6 +696,24 @@ export class FieldMetadataService extends TypeOrmQueryService<FieldMetadataEntit
     return translatedMessage;
   }
 
+  private prepareCustomFieldMetadataOptions(
+    options?: FieldMetadataDefaultOption[] | FieldMetadataComplexOption[],
```

Comment:
I would force options to be defined and not call it in the parent if options is undefined. This simplifies the logic usually and keep the branching tree shorter

Reply:
I wanted to simplify the spread operation from the caller but indeed I agree about to review and see what I can do here